src/plugins/plugins-list.js

```diff
@@ -74,3 +76,23 @@ export const presets = {
     defaultConfig: {},
   },
 };
+
+const pluginMap = new Map([
```

Comment:
Any reason for using `Map` instead of just doing this?
```js
{
  "plugin-name": "pluginName",
  //...
}
```

Reply:
No concrete reason other than just using key-value pairs! (: